openapi/src/parts/tps.yaml

```diff
@@ -68,6 +68,10 @@ components:
           uniqueItems: true
           items:
             $ref: '#/components/schemas/WsmPolicyObjectId'
+        createdDate:
+          type: string
```

Comment:
Do we want to give these the `format: date-time` like we did in WSM?

Reply:
No, I don't want to do that. The data is already in string form coming out of TPS, so it seems useless to parse and convert it just to convert it back again.

Reply:
what does the client-side output looks like? my only concern is that ui need to convert it?

Reply:
It is the standard ISO 8601 format. I'll add that to the documentation.